linchpin/defaults/schemas/schema_v4.json

```diff
@@ -108,8 +108,35 @@
                                  "keyvalue": {
                                      "type": "array"
                                  },
-                                 "hostrequire": {
-                                     "type": "array"
+                                 "hostrequires": {
+                                     "type": "array",
+                                     "items": {
+                                         "not": {
+                                             "allOf": [
+                                                 {"required": ["force", "tag"]}
+                                             ]
+                                         },
```

Comment:
schemacheck looks perfect, I'll run some examples through the schema and see how I did. :)